CHANGELOG.md

```diff
@@ -1,3 +1,247 @@
+- `select_line_above` / `select_line_below` - extend or shrink a selection based on the direction and anchor (#9080)
+
+Usability improvements:
+
+- Default `roots` key of `[[language]]` entries in `languages.toml` configuration (#8803)
+- Exit select mode in commands that modify the buffer (#8689)
+- Use crossterm cursor when out of focus (#6858, #8934)
+- Join empty lines with only one space in `join_selections` (#8989)
+- Allow configuring the indentation heuristic (#8307)
+- Check for LSP rename support before showing rename prompt (#9277)
+- Normalize `S-<lower-ascii>` keymaps to uppercase ascii (#9213)
+- Add formatter status to `--health` output (#7986)
+- Change path normalization strategy to not resolve symlinks (#9330)
+- Select subtree within injections in `:tree-sitter-subtree` (#9309)
+- Use tilde expansion and normalization for `$HELIX_RUNTIME` paths (1bc7aac)
+- Improve failure message for LSP goto references (#9382)
+- Use injection syntax trees for bracket matching (5e0b3cc)
+- Respect injections in `:tree-sitter-highlight-name` (8b6565c)
+- Respect injections in `move_parent_node_end` (035b8ea)
+- Use `gix` pipeline filter instead of manual CRLF implementation (#9503)
+- Follow Neovim's truecolor detection (#9577)
+- Allow specifying required roots for situational LSP activation (#8696)
+- Reload language configuration with `:reload`, SIGHUP (#9415)
+- Allow numbers as bindings (#8471, #9887)
+- Exit prompts with backspace when the prompt is empty (#9828)
+- Respect undercurl config when terminfo is not available (#9897)
+- Ignore `.pijul`, `.hg`, `.jj` in addition to `.git` in file pickers configured to show hidden files (#9935)
+- Add completion for registers to `:clear-register` and `:yank-diagnostic` (#9936)
+- Repeat last motion for goto next/prev diagnostic (#9966)
+- Allow configuring a character to use when rendering narrow no-breaking space (#9604)
+
+Fixes:
+
+- Use terminfo to reset terminal cursor style (#8591)
+- Fix precedence of `@align` captures in indentat computation (#8659)
+- Only render the preview if a Picker has a preview function (#8667)
+- Fix the precedence of `ui.virtual.whitespace` (#8750, #8879)
+- Fix crash in `:indent-style` (#9087)
+- Fix `didSave` text inclusion when sync capability is a kind variant (#9101)
+- Don't automatically dismiss zero width diagnostics (#9280)
+- Update the history of newly focused views (#9271)
+- Initialize diagnostics when opening a document (#8873)
+- Sync views when applying edits to unfocused views (#9173)
+- Treat non-numeric `+arg`s passed in the CLI args as filenames (#9333)
+- Fix crash when using `mm` on an empty plaintext file (2fb7e50)
+- Ignore empty tree-sitter nodes in match bracket (445f7a2)
+- Exit a language server if it sends a message with invalid JSON (#9332)
+- Handle failures to enable bracketed paste (#9353)
+- Gate all captures in a pattern behind `#is-not? local` predicates (#9390)
+- Make path changes LSP spec conformant (#8949)
```

Comment:
You may want to mention this together with the move command since this really does all the heavy lifting to make the move commands lsp support work (it was basically completely broken before this PR) I think that should be mentioned there aswell